pkg/envoy/ads/server.go

```diff
@@ -85,7 +85,5 @@ func (s *Server) Start(ctx context.Context, cancel context.CancelFunc, port int)
 		return fmt.Errorf("error starting ADS server: %w", err)
 	}
 
-	s.ready = true
```

Comment:
It looks like "readiness" wasn't marked ready until the ADS server was up and running.  Since that code is run before the httpServer and returns a fatal event.  This should be fine.